lte/gateway/configs/templates/mme.conf.template

```diff
@@ -263,8 +263,19 @@ MME :
     };
 
     S-GW :
-   {
-    # S-GW binded interface for S11 communication (GTPV2-C), if none selected the ITTI message interface is used
-       SGW_IPV4_ADDRESS_FOR_S11              = "{{ remote_sgw_ip }}";
-   };
+    {
+        # S-GW binded interface for S11 communication (GTPV2-C), if none selected the ITTI message interface is used
+        SGW_IPV4_ADDRESS_FOR_S11              = "{{ remote_sgw_ip }}";
+    };
+
+    FEDERATED_MODE_MAP = (
+        {% for fed_map in federated_mode_map -%}
+        {
+          MODE        = "{{ fed_map.mode }}"
```

Comment:
It would be good to add a comment here to specify the mapping of integer values to the given federated mode here

Reply:
yep, per our discussion is in lte `mconfig.proto`

```
    FederatedModeMap federated_mode_map = 34;
}
message FederatedModeMap {
    bool enabled = 1;
    repeated ModeMapItem mapping = 2;
}

```

Reply:
I wanted to avoid any hardcoded value in the config file. What about of referencing to lte mconfig.proto to check the values. 

The reason is if we hardcode the comments here, the day we change those values on the proto we will not remember to change those here.